src/components/ColumnSettingsList.ts

```diff
@@ -223,7 +223,7 @@ export const ColumnSettingsList = ({ items, onChange, toggleVisibility }: Column
     const item = _.filter({ id })(items) as unknown;
     return item ? item[0].name : 'unknown';
   };
-  const announcements = <Announcements>{
+  const announcements: Announcements = {
```

Comment:
`yarn storybook` was failing after I upgraded Storybook because of this syntax (babel was producing an error).